uberfire-extensions/uberfire-experimental-extensions/uberfire-experimental-extensions-client/src/main/resources/org/uberfire/ext/experimental/client/resources/i18n/UberfireExperimentalConstants.properties

```diff
@@ -0,0 +1,3 @@
+ExperimentalFeaturesEditorScreenViewImpl.experimentalFeaturesTitle=Experimental Features
+ExperimentalFeaturesEditorScreenViewImpl.experimentalFeaturesWarning=This are experimental features that are under development, \
```

Comment:
This -> These
on a productive environment -> in production environment

Reply:
My english...